php_phongo.c

```diff
@@ -864,10 +907,10 @@ void php_phongo_read_preference_to_zval(zval *retval, const mongoc_read_prefs_t
 	}
 
 	if (!bson_empty0(tags)) {
+		/* Use PHONGO_TYPEMAP_NATIVE_ARRAY for the root type since tags is an
+		 * array; however, inner documents and arrays can use the default. */
```

Comment:
`PHONGO_BSON_STATE_INITIALIZER` is already the default, which we use for the `state` intialization. This should suffice.